pprof.go

```diff
@@ -19,13 +19,85 @@ package main
 import (
 	"fmt"
 	"os"
+	"strings"
 
+	"github.com/chzyer/readline"
 	"github.com/google/pprof/driver"
 )
 
 func main() {
-	if err := driver.PProf(&driver.Options{}); err != nil {
+	if err := driver.PProf(&driver.Options{UI: newUI()}); err != nil {
 		fmt.Fprintf(os.Stderr, "pprof: %v\n", err)
 		os.Exit(2)
 	}
 }
+
+// readlineUI implements the driver.UI interface using the
+// github.com/chzyer/readline library
```

Comment:
I'd add a sentence to summarize what we discussed which is "It is contained in pprof.go to avoid the readline dependency in the vendored copy of pprof in Go distribution (which does not use this file)".